crates/rolldown/src/module_finalizers/scope_hoisting/mod.rs

```diff
@@ -467,6 +468,61 @@ impl<'me, 'ast> ScopeHoistingFinalizer<'me, 'ast> {
         }
         _ => {}
       }
+      // rewrite `import.meta.ROLLUP_FILE_URL_referenceId`
+      if let Some(reference_id) = property_name.strip_prefix("ROLLUP_FILE_URL_") {
```

Comment:
Can you extract this into a function, so that we don't need   `#[allow(clippy::too_many_lines)]`